src/etl/LoadBalancer.hpp

```diff
@@ -44,8 +45,10 @@
 #include <xrpl/proto/org/xrpl/rpc/v1/xrp_ledger.grpc.pb.h>
 
 #include <chrono>
+#include <concepts>
 #include <cstdint>
 #include <expected>
+#include <functional>
```

Comment:
These includes look unrelated to the change

Reply:
Clangd said they were missing, so I added. I don't think they make any difference.